slick.grid.js

```diff
@@ -347,176 +344,198 @@ if (typeof Slick === "undefined") {
+
+      _headers.forEach((el) => {
+        u.width(el, getHeadersWidth());
+      })
+
+      u.width(_headerRowSpacerL, getCanvasWidth() + scrollbarDimensions.width);
+      u.width(_headerRowSpacerR, getCanvasWidth() + scrollbarDimensions.width);
 
       // footer Row
       if (options.createFooterRow) {
-        $footerRowScrollerR = $("<div class='slick-footerrow ui-state-default' />").appendTo($paneTopR);
-        $footerRowScrollerL = $("<div class='slick-footerrow ui-state-default' />").appendTo($paneTopL);
+        _footerRowScrollerR = u.template("<div class='slick-footerrow ui-state-default' />", _paneTopR);
+        _footerRowScrollerL = u.template("<div class='slick-footerrow ui-state-default' />", _paneTopL);
 
-        $footerRowScroller = $().add($footerRowScrollerL).add($footerRowScrollerR);
+        _footerRowScroller = [_footerRowScrollerL, _footerRowScrollerR];
 
-        $footerRowSpacerL = $("<div style='display:block;height:1px;position:absolute;top:0;left:0;'></div>")
-          .css("width", getCanvasWidth() + scrollbarDimensions.width + "px")
-          .appendTo($footerRowScrollerL);
-        $footerRowSpacerR = $("<div style='display:block;height:1px;position:absolute;top:0;left:0;'></div>")
-          .css("width", getCanvasWidth() + scrollbarDimensions.width + "px")
-          .appendTo($footerRowScrollerR);
+        _footerRowSpacerL = u.template("<div style='display:block;height:1px;position:absolute;top:0;left:0;'></div>", _footerRowScrollerL);
+        u.width(_footerRowSpacerL, getCanvasWidth() + scrollbarDimensions.width);
+        _footerRowSpacerR = u.template("<div style='display:block;height:1px;position:absolute;top:0;left:0;'></div>", _footerRowScrollerR);
+        u.width(_footerRowSpacerR, getCanvasWidth() + scrollbarDimensions.width);
 
 
-        $footerRowL = $("<div class='slick-footerrow-columns slick-footerrow-columns-left' />").appendTo($footerRowScrollerL);
-        $footerRowR = $("<div class='slick-footerrow-columns slick-footerrow-columns-right' />").appendTo($footerRowScrollerR);
+        _footerRowL = u.template("<div class='slick-footerrow-columns slick-footerrow-columns-left' />", _footerRowScrollerL);
+        _footerRowR = u.template("<div class='slick-footerrow-columns slick-footerrow-columns-right' />", _footerRowScrollerR);
 
-        $footerRow = $().add($footerRowL).add($footerRowR);
+        _footerRow = [_footerRowL, _footerRowR];
 
         if (!options.showFooterRow) {
-          $footerRowScroller.hide();
+          _footerRowScroller.forEach((scroller) => {
+            hide(scroller);
+          });
         }
       }
 
-      $focusSink2 = $focusSink.clone().appendTo($container);
+      _focusSink2 = _focusSink.cloneNode(true);
+      _container.append(_focusSink2);
 
       if (!options.explicitInitialization) {
         finishInitialization();
       }
     }
 
+    function show(el, type)
+    {
```

Comment:
could you use the JS standard and move the opening curly brace on the same line as the function  `function () {`

Reply:
I'm not sure why you marked this as resolved, what I'm asking to lower the number of lines 

from this
```js
function show(el, type)
{ 
```
to this 
```js
function show(el, type) {
```
for the entire file and all other files

Reply:
Sorry for that, I made the changed and reverted it. Auto-formatting the js files created a lot of noise. I will do the formatting at the end, when everything is working. Is this okay?

Reply:
sure, as long as it's done eventually ;)